test/geneexpr_test.jl

```diff
@@ -12,7 +12,7 @@ dotestmean = true
 doprintmeans = false
 
 # SSAs to test
-SSAalgs = (RDirect(), RSSACR(), Direct(), DirectFW(), FRM(), FRMFW(), SortingDirect(),
+SSAalgs = (JumpProcesses.NullAggregator(), RDirect(), RSSACR(), Direct(), DirectFW(), FRM(), FRMFW(), SortingDirect(),
```

Comment:
Would it make sense to, for completeness, have a test calling `JumpProblem(prob, jumpset)`, and not always going via `JumpProcesses.NullAggregator()`? Like, it totally should be fine, but this is probably how most users will en up calling `JumpProblem`s, so it might be prudent to have that version in the tests as well.

Reply:
Added at the end of this file.